src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
```

Comment:
Kindly import the two reducers you implemented in the redux directory for them to be accesible by the store; don't forget to configure them into the reducer.

```Example
import { configureStore } from "@reduxjs/toolkit"
import booksReducer from "./books/booksSlice"
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoryReducer },
})
;
```

The code snippet is just an example.